APE24.rst

```diff
@@ -0,0 +1,477 @@
+   (also see **"Re-review" of packages**). We will encourage these older
+   packages that are still actively maintained to also go through the
+   new process, pointing them to the benefits mentioned in
+   **What do we gain from a partnership with pyOpenSci?**
+4. For the new packages, have our project website ingest a RSS/XML feed
+   from pyOpenSci for cross-listing (also see **Listing on websites**).
+   We would also link back to pyOpenSci on our page.
+   Meanwhile, we would keep the pre-APE 24 listing on a "legacy affiliation" page
+   that will be kept alive during the lifetime of the Astropy project.
+5. Work with pyOpenSci to cross-list Affiliated editors, between their
+   editors listing and our roles page. For example, pyOpenSci could
+   mark Astropy editors on the `pyOpenSci Editorial Board`_ with Astropy logo.
+6. Work with pyOpenSci to hash out a more concrete plan on how to
+   get a previously sunsetted package to be listed actively again
+   if the package is revived.
+7. Work with pyOpenSci to come up with a process to swap Affiliated
+   editors in and out of `pyOpenSci Editorial Board`_, since our
+   Affiliated editor assignment is not permanent.
+
+The goal is to have a migration that is not too disruptive to current process.
+While this migration could take years to complete, the turnaround time
+for any one package should be much shorter (months, perhaps).
+
+The trial period
+----------------
+
+**Note: Communication is very important at this stage!**
+
+This trial is only done for new Affiliated requests that are early enough
+in the process as not to have work duplicated for package maintainers,
+reviewers, and editors.
+
+Editors will give qualifying packages the option to try out this new process.
+Regardless of the outcome of this APE, if the package is accepted during this trial,
+the acceptance stands; That is, if this APE falls through but the package used
+the proposed process here, it still counts as Affiliated and does not have to re-apply.
+
+Reviewers who agree to participate will use the `reviewer sign-up form`_
+for pyOpenSci. If a package agrees but the chosen reviewer declines this trial,
+the editor assigned will find a new reviewer.
+
+
+Backward compatibility
+======================
+
+.. This section describes the ways in which the APE breaks backward compatibility.
+
+Somewhat compatible:
+
+* Packaging guidelines are pretty similar.
+* Existing Affiliated editors would join the `pyOpenSci Editorial Board`_.
+* There would be cross-listing between Affiliated editors over at pyOpenSci and our roles page.
+* There would be cross-listing between Affiliated packages over at pyOpenSci
+  and Affiliated page.
+
+Not backward compatible:
+
+* Reviewers can no longer be anonymous and have to sign up via `reviewer sign-up form`_.
+  The whole review process is open.
+* Instead of static color badges, there will be a link to full pyOpenSci
+  review for that package.
```

Comment:
So as we are also planning to do the same as Astropy and partner with PyOpenSci, @Cadair pointed out that badges give even a very uniformed user a good impression of the quality of the packages.

Whereas on the surface the PyOpenSci review while more complex does not offer a glance of the overall quality from the review page.

Does anyone from Astropy have thoughts on this, at a glance ability?

Reply:
Sure the badges being outdated is a problem but in a world where we could automate those, is there no attachment to them on the astropy side?

Reply:
> Is there no desire from the Astropy side for the simple at a glance nature the badges could provide?

There is. But not part of the initial rollout. Once we have the automation, we could find a way to grab the new badges and re-display on Astropy page.

Reply:
this is a great conversation. automating such a badge is something we've been talking about. My thought here (let me know what y'all think) is that we first get our partnership process solidified. Currently, we offer a pyOpenSci badge indicating a package has been reviewed / accepted.

However, in the future we can provide a server hosted badge (rOpenSci does this) that we can change the color of via a ci build that checks x,y,z metrics. For instance, some indication of maintenance status measured by repo activity ([see pr here on archiving projects)](https://github.com/pyOpenSci/software-peer-review/pull/258). I think we should collectively - astropy, sunpy, pyhc, scverse etc come together and discuss what types of metrics are important to us and what data would be used to capture such metrics. How fragile the data are (ie a ci build badge might change locations over time whereas github use metrics for a repo will be consistent to collect). Then we can decide how to show current status. 

Let's work on this as a community and then we can find funding to implement this work / building upon and learning from any existing efforts in this space as it makes sense.